Code from Zhu

charleyjoo opened the pull request:
Uploaded the code to merge elevation into road segments

charleyjoo committed:
Code from Zhu

Uploaded the code to merge elevation into road segments

johnash90 commented on crashDataAnalysisTools/merge_elevation_and_segment.py:
```diff
@@ -0,0 +1,57 @@
+import os
+
```
This space should be removed.

johnash90 commented on crashDataAnalysisTools/merge_elevation_and_segment.py:
```diff
@@ -0,0 +1,57 @@
+
+import pandas as pd
+import sqlite3 as dbi
+
```
Need one more blank space here to be pep8 compliant.

johnash90 commented on crashDataAnalysisTools/merge_elevation_and_segment.py:
```diff
@@ -0,0 +1,57 @@
+    cu.execute('DROP TABLE IF EXISTS roads')
+    cu.execute('DROP TABLE IF EXISTS elev')
+
+    # convert the pandas datafromes into database tables through the connection
```
Spelling error, should be 'dataframes'

johnash90 commented on crashDataAnalysisTools/merge_elevation_and_segment.py:
```diff
@@ -0,0 +1,57 @@
+    roads.to_sql(name='roads', con=conn)
+    elev.to_sql(name='elev', con=conn)
+
+    # sql query statement to summarize and merge elevation data
```
It may help to provide a line or two of comment explaining what the query does instead of being so general.

johnash90 commented on crashDataAnalysisTools/merge_elevation_and_segment.py:
```diff
@@ -0,0 +1,57 @@
+    elev.to_sql(name='elev', con=conn)
+
+    # sql query statement to summarize and merge elevation data
+    query_merge_elev = '''
```
In the end we may find it more useful to put all of our sql queries in one file that we can import; let's talk about this later.

charleyjoo replied:
Definately

johnash90 commented on crashDataAnalysisTools/merge_elevation_and_segment.py:
```diff
@@ -0,0 +1,57 @@
+import os
```
It does not appear as if this module is used right now.

charleyjoo committed:
Update merge_elevation_and_segment.py

Summary of changes
- Removed `import os` cuz it's not applied in the code
- Added one more blank line before the function
- Added more descriptions of the SQL statement
- Corrected some typos
- Updated some function and variable descriptions
